modular_zubbers/code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -0,0 +1,5 @@
+/obj/structure/closet/examine(mob/user)
+	. = ..()
+	if(isobserver(user))
+		. += span_info("It contains: [english_list(contents)].")
+		investigate_log("had its contents examined by [user] as a ghost.", INVESTIGATE_GHOST)
```

Comment:
Do we really want an investigate log generated every time someone examines a locker?

Reply:
Alright, it's removed.